genai-perf/tests/test_exporters/test_console_exporter.py

```diff
@@ -454,20 +432,22 @@ def test_valid_telemetry_verbose(self, monkeypatch, capsys) -> None:
         exporter.export(width=100)  # fix width for consistent output
 
         expected_content = (
-            "                           NVIDIA GenAI-Perf | LLM Metrics                            \n"
-            "┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━┳━━━━━━━┳━━━━━━━┳━━━━━━━┳━━━━━━━┳━━━━━━━┓\n"
-            "┃                         Statistic ┃    avg ┃   min ┃   max ┃   p99 ┃   p90 ┃   p75 ┃\n"
-            "┡━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━╇━━━━━━━╇━━━━━━━╇━━━━━━━╇━━━━━━━╇━━━━━━━┩\n"
-            "│          Time To First Token (ms) │   8.00 │  7.00 │  9.00 │  8.98 │  8.80 │  8.50 │\n"
-            "│         Time To Second Token (ms) │   2.00 │  1.00 │  3.00 │  2.98 │  2.80 │  2.50 │\n"
-            "│              Request Latency (ms) │   5.00 │  4.00 │  6.00 │  5.98 │  5.80 │  5.50 │\n"
-            "│          Inter Token Latency (ms) │  11.00 │ 10.00 │ 12.00 │ 11.98 │ 11.80 │ 11.50 │\n"
-            "│   Output Sequence Length (tokens) │   2.00 │  1.00 │  3.00 │  2.98 │  2.80 │  2.50 │\n"
-            "│    Input Sequence Length (tokens) │   6.00 │  5.00 │  7.00 │  6.98 │  6.80 │  6.50 │\n"
-            "│ Output Token Throughput (per sec) │ 456.00 │   N/A │   N/A │   N/A │   N/A │   N/A │\n"
-            "│      Request Throughput (per sec) │ 123.00 │   N/A │   N/A │   N/A │   N/A │   N/A │\n"
-            "│             Request Count (count) │   3.00 │   N/A │   N/A │   N/A │   N/A │   N/A │\n"
-            "└───────────────────────────────────┴────────┴───────┴───────┴───────┴───────┴───────┘\n"
+            "                             NVIDIA GenAI-Perf | LLM Metrics                             \n"
+            "┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━┳━━━━━━━┳━━━━━━━┳━━━━━━━┳━━━━━━━┳━━━━━━━┓\n"
+            "┃                            Statistic ┃    avg ┃   min ┃   max ┃   p99 ┃   p90 ┃   p75 ┃\n"
+            "┡━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━╇━━━━━━━╇━━━━━━━╇━━━━━━━╇━━━━━━━╇━━━━━━━┩\n"
+            "│             Time To First Token (ms) │   8.00 │  7.00 │  9.00 │  8.98 │  8.80 │  8.50 │\n"
+            "│            Time To Second Token (ms) │   2.00 │  1.00 │  3.00 │  2.98 │  2.80 │  2.50 │\n"
+            "│                 Request Latency (ms) │   5.00 │  4.00 │  6.00 │  5.98 │  5.80 │  5.50 │\n"
+            "│             Inter Token Latency (ms) │  11.00 │ 10.00 │ 12.00 │ 11.98 │ 11.80 │ 11.50 │\n"
+            "│     Output Token Throughput Per User │   5.00 │  4.00 │  6.00 │  5.98 │  5.80 │  5.50 │\n"
```

Comment:
Is this correct? Same with the above screenshots. These values match the request latencies exactly. And TTST == OSL.

Reply:
Yes, the values match because we are assigning the same (dummy) values to the metric (e.g. `[4, 5, 6]`). This is okay because the purpose of these tests are to check if the console is properly displaying the passed values. The calculation itself is being checked by other tests.